### PR TITLE
Update platform list

### PR DESCRIPTION
**Pull request by kroeckx**

### CODE REVIEW

File: policies/platformpolicy.html [hunk 1]
```diff
@@ -205,6 +205,1317 @@ <h1>Platform Policy</h1>
                   </tr>
                 </table>
               </p>
+              <p>
+                The current unadopted platforms are:
+
+                <table summary="Community Platforms">
```
Comment (t-j-h) [inline policies/platformpolicy.html:211]: They are not "Community Platforms" - title should be "Unadopted Platforms"

**Comment by mattcaswell** [comment]
While we're making fixes could we also fix the items noted in the meeting minutes:
https://git.openssl.org/gitweb/?p=otc.git;a=blob;f=meeting-minutes/minutes-2021-10-19.txt;h=8bae2b86ecd7c4f967ba2aa822535dc0facbbfa9;hb=HEAD

"Matt presented the updates that have been made to the platform policy. Richard noted that the OpenVMS version is wrong for Itanium. Should be 8.4. FreeBSD clang version 11.0.1 is default for FreeBSD 13.0"

Or I can do this in a separate PR if you prefer.

File: policies/platformpolicy.html (hunk 1 above)
Comment (kroeckx) [reply policies/platformpolicy.html:211]: It says so on line 209?

**Comment by kroeckx** [comment]
I pushed this to the wrong place

File: policies/platformpolicy.html (hunk 1 above)
Comment (mattcaswell) [reply policies/platformpolicy.html:211]: But on 211 it says Community.

**Comment by t-j-h** [comment]
Marking this as approved - given #268 fixes the one comment - and it has been pushed.
So retrospective approval ...

**Pull request merged**